### ISSUE TITLE
conv size of up-conv2x2

### ISSUE THREAD

**Issue by JohnTargaryen** [LeeJunHyun/Image_Segmentation#35]
Hi, the paper says that "Every step in the expansive path consists of an upsampling of the
feature map followed by a 2x2 convolution ('up-convolution') that halves the
number of feature channels"
1. In your code, however, I notice that you use a conv2d of size3x3, would you explain the reason to that?
```py
class up_conv(nn.Module):
    def __init__(self,ch_in,ch_out):
        super(up_conv,self).__init__()
        self.up = nn.Sequential(
            nn.Upsample(scale_factor=2),
            nn.Conv2d(ch_in,ch_out,kernel_size=3,stride=1,padding=1,bias=True),
		    nn.BatchNorm2d(ch_out),
			nn.ReLU(inplace=True)
        )

    def forward(self,x):
        x = self.up(x)
        return x
```

2. And also, let's say I want to stick to the paper and use 2x2 conv, how would I choose the padding and stride to make output size the same as the input size?

Thanks for sharing your work.

**Comment by LeeJunHyun** [comment]
Hi, @JohnTargaryen.
Thank you for your interest in my repo.
I just had wanted to compare a series of U-Net.
So I adapted a same convolution filter size to models.
If you want, please modify the convolution filter size.
You can calculate the padding and the stride size by referring [here](https://adeshpande3.github.io/A-Beginner%27s-Guide-To-Understanding-Convolutional-Neural-Networks-Part-2/).
Thanks.

**Comment by JohnTargaryen** [comment]
> Hi, @JohnTargaryen.
> Thank you for your interest in my repo.
> I just had wanted to compare a series of U-Net.
> So I adapted a same convolution filter size to models.
> If you want, please modify the convolution filter size.
> You can calculate the padding and the stride size by referring [here](https://adeshpande3.github.io/A-Beginner%27s-Guide-To-Understanding-Convolutional-Neural-Networks-Part-2/).
> Thanks.

Thx, I'll look into that
